src/components/input/input.e2e.ts

```diff
@@ -1350,25 +1359,139 @@ describe("calcite-input", () => {
     expect(await element.getProperty("value")).toBe("-123");
   });
 
-  describe("is form-associated", () => {
-    it("supports type=text", () => formAssociated("calcite-input", { testValue: "test" }));
-    it("supports type=number", () => formAssociated("<calcite-input type='number'></calcite-input>", { testValue: 5 }));
-  });
+  describe("ArrowUp/ArrowDown function as Home/End within calcite-input", () => {
+    let page;
 
-  describe("external/programmatic changes to the value", () => {
-    it("incrementing correctly updates the value after focus and blur events", async () => {
-      const page = await newE2EPage();
-      await page.setContent(html` <calcite-input type="number" value="1"></calcite-input> `);
+    beforeEach(async () => {
+      page = await newE2EPage();
+    });
+
+    it("works for type textarea", async () => {
+      await page.setContent(`<calcite-input id="text-element" type="textarea" ></calcite-input>`);
```

Comment:
If the `id` isn't referenced, let's drop it. This applies to other HTML snippets.